pro_tes/middleware/middleware.py

```diff
@@ -29,7 +29,7 @@ class TaskDistributionMiddleware(AbstractMiddleware):
 
     Attributes:
         tes_uri: TES instance best suited for TES task.
-        input_uris: A list of input URIs in the incoming request.
+        input_uris: [...] from the incoming request.
```

Comment:
Sorry, I meant "A list of input URIs from the incoming request" :)

[...] I used as a placeholder for everything that came before the "from" ;-)

Reply:
Oh, I should have thought that, will fix in the next commits

Reply:
Fixed